types/common.ts

```diff
@@ -2,7 +2,27 @@
  * TODO: implement the type
  * https://github.com/ConnecMent/dydx-bot/issues/1
  */
-export type Candle = any;
 
 export type Pair = string;
-export type PositionType = "long" | "short" | null;
+export type PositionType = 'long' | 'short' | null;
+export type TimeFrame = string;
```

Comment:
Is `TimeFrame` and `resolution` of `Candle` the same? If so, you should update `TimeFrame` and reuse it in `Candle`. If not, just ignore this comment.

Reply:
yes they are same . I fixed that 
when I test every value we set for timeFrame exactly the same value appear for the resolution